packages/notebook-extension/src/index.ts

```diff
@@ -3564,7 +3564,7 @@ function addCommands(
 
   // All commands with isEnabled defined directly or in a semantic commands
   // To simplify here we added all commands as most of them have isEnabled
-  const skip = [CommandIDs.createNew];
+  const skip = [CommandIDs.createNew, CommandIDs.createOutputView];
   const notify = () => {
     Object.values(CommandIDs)
       .filter(id => !skip.includes(id))
```

Comment:
Could we call `app.commands.hasCommand(id)` here? Something like:

```suggestion
      .filter(id => !skip.includes(id) && app.commands.hasCommand(id))
```

Ref: https://lumino.readthedocs.io/en/latest/api/classes/commands.CommandRegistry-1.html#hasCommand

Would it be a full fix?

Reply:
@jtpio what do you think? I would like to release a new beta soon, but I do not want to do this without unblocking you on Notebook with this.

Reply:
Hmm I think the full fix would be to only notify the commands added by the plugin. But that would require keeping track of the list of commands and iterate through them in this loop to call `notifyCommanChanged`. Which is maybe why this code chooses to notify all of the them instead.

Reply:
But wouldn't we still want to notify commands added by other plugins (if these plugins are active)?